src/scripts/import/controllers/ImportController.js

```diff
@@ -0,0 +1,27 @@
+module.exports = /* @ngInject */function (importItems, translationPrefix, defaultValues, fileExample) {
+  var controller = this
+  controller.translationPrefix = translationPrefix
+  controller.defaultValues = defaultValues
+  controller.fileExample = fileExample
+  controller.importing = false
+  controller.importData = {
+    create: false,
+    update: false,
+    failOnError: false,
+    dryRun: false,
+    defaults: {}
+  }
+
+  controller.import = function () {
+    controller.importing = true
+    importItems(Object.assign({}, controller.importData, {defaults: JSON.stringify(controller.importData.defaults)}))
```

Comment:
This makes `controller.importData.defaults` always replaced by the stringified version. Isn't it better to just move them out of `importData` structure?